src/gui/guiPathSelectMenu.h

```diff
@@ -44,6 +44,10 @@ class GUIFileSelectMenu : public GUIModalMenu
 
 	void setTextDest(TextDest *dest) { m_text_dst = dest; }
 
+protected:
+	std::wstring getLabelByID(s32 id) { return L""; }
```

Comment:
as many values are by default identical, i think it's better to have them in parent class and override in child classes

Reply:
These methods are pure virtual out of necessity so must be overridden by every derived class. I didn't really like doing that either but cannot see any easier way around it.

Reply:
Seems like we need another opinion on this, @sfan5 maybe?

Reply:
@nerzhul maybe this isn't a big issue and could be dealt with later if necessary?

Reply:
@stujones11 lgtm